sdk/monitor/monitor-ingestion/samples-dev/maxConcurrency.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Demonstrates how to control the number of concurrent requests using the maxConcurrency option
+ */
+
+import { DefaultAzureCredential } from "@azure/identity";
+import { LogsIngestionClient } from "@azure/monitor-ingestion";
+
+require("dotenv").config();
+
+async function main() {
+  const logsIngestionEndpoint = process.env.LOGS_INGESTION_ENDPOINT || "logs_ingestion_endpoint";
+  const ruleId = process.env.DATA_COLLECTION_RULE_ID || "data_collection_rule_id";
+  const streamName = process.env.STREAM_NAME || "data_stream_name";
+  const credential = new DefaultAzureCredential();
+  const client = new LogsIngestionClient(logsIngestionEndpoint, credential);
+
+  // Constructing a large number of logs to ensure batching takes place
+  const logs = [];
+  for (let i = 0; i < 100000; ++i) {
+    logs.push({
+      Time: "2021-12-08T23:51:14.1104269Z",
+      Computer: "Computer1",
+      AdditionalContext: `context-${i}`,
+    });
+  }
+
+  // Set the maximum concurrency to 1 to prevent concurrent requests entirely
+  const result = await client.upload(ruleId, streamName, logs, { maxConcurrency: 1 });
```

Comment:
do we need another sample showing parallel requests too?

Reply:
We can add another sample, wouldn't hurt. Or it might be enough to just add a bit more comments to this sample (and in the README) about when you'd want to control the level of parallelism, beyond just the example of disabling it entirely?